timetable/utils.py

```diff
@@ -44,11 +44,14 @@ def get_timetable_context(lessons):
     periods = [period.number for period in default_periods]
     period_strs = get_period_strings(default_periods)
 
+    alternative_periods = Period.objects.filter(schedule=2)
```

Comment:
I don't think you should hardcode stuff like this. What if for example school will have 5 different schedules?

Reply:
Okay, I will take a look at it.

Reply:
@Igifigi I have fixed it so it works with multiple alternative schedules.